### PR TITLE
[ADD][10.0] account_asset migration script

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: addons/account_asset/migrations/10.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+column_renames = {
+    'account_asset': [
+        ('asset_id', None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
```
Comment (pedrobaeza) [inline addons/account_asset/migrations/10.0.1.0/pre-migration.py]: This is env now by default and you have to use `env.cr` then

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/account_asset/migrations/10.0.1.0/pre-migration.py (hunk 1 above)
Comment (cubells) [reply addons/account_asset/migrations/10.0.1.0/pre-migration.py]: Do you prefer to use use_env=False or to use use_env=True?

Comment (pedrobaeza) [reply addons/account_asset/migrations/10.0.1.0/pre-migration.py]: I prefer the standard (which is use_env=True)

File: addons/account_asset/migrations/10.0.1.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+column_renames = {
+    'account_move': [
+        ('asset_id', None),
+    ],
+}
+
+
+@openupgrade.migrate(use_env=True)
```
Comment (pedrobaeza) [inline addons/account_asset/migrations/10.0.1.0/pre-migration.py:15]: No need to put `use_env=True`, as it's the default in v10, but it's not so important.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**